docs/api-docs/webhooks/webhook-events.mdx

```diff
@@ -773,7 +828,12 @@ Payload objects with the following scopes take the form that follows:
   "store_id": "1025646",
   "data": {
     "type": "product", // will always be product
-    "id": 205 // ID of the product
+    "id": 205, // ID of the product
+    "attributes": ["warranty", "is_featured"],
```

Comment:
I assume the `attributes` and `context` fields only apply to the `store/product/updated` scope, and not the `store/product/deleted` and `store/product/created` scope?

Reply:
Changed the doc so that the payload for the `store/product/updated` scope is different than that of the `store/product/deleted` and `store/product/created` scope (due to comment [here](https://github.com/bigcommerce/docs/pull/252#discussion_r1605135873))